app/src/main/java/rick_and_morty/data/event/UIState.kt

```diff
@@ -0,0 +1,10 @@
+package rick_and_morty.data.event
+
+import rick_and_morty.data.model.CharacterResultsDto
+
+data class UIState (
+    var isLoading : Boolean = false,
+    val isSuccess: List<CharacterResultsDto> = emptyList(),
```

Comment:
this name is a bit misleading....

Reply:
@LordRaydenMK do you think "characterResult" is better?